### PR TITLE
Added KI for the MySQL.Data.dll crash

### PR DESCRIPTION
**Pull request by BrentSLC**
Added KI for MySQL.Data.dll crash to be used as jumping off point for analyzers and other docs

### CODE REVIEW

File: user-guide/Troubleshooting/Known_issues/KI_MySQL_Unhandled_Exception.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+uid: KI_MySQL_Unhandled_Exception
+---
+
+# MySQL.Data.dll can cause service crashes
```
Comment (MariekeGO) [inline user-guide/Troubleshooting/Known_issues/KI_MySQL_Unhandled_Exception.md:5]: Could you rephrase this title so it is more a description of the issue than of the cause? With the current title, people who go looking for the issue based on what they encounter in their system may not know that this is the page they are looking for.

Comment (BrentSLC) [reply user-guide/Troubleshooting/Known_issues/KI_MySQL_Unhandled_Exception.md]: Will do.

**Comment by MariekeGO** [comment]
@BrentSLC, I've reviewed your PR and made some changes. Could you check if everything is still OK for you? 
Also, I noticed that RN 40200 seemed to be related to this issue. Is this the RN that will introduce the fix for this?

**Review by BrentSLC — COMMENTED** [review]
Looks good to me.
Yes RN 40200 is indeed related. That RN should prevent SLDataGateway crashes due to this issue (Connectors can use their own MySQL.Data.dll, so those will remain at risk of the crashes.)

**Review by MariekeGO — APPROVED** [review]
